locales/en-US/app.l20n

```diff
@@ -95,3 +95,7 @@
 <tourDoneButton "Done">
 <userCountContainer "There are <span class=\"bold\">{{installation_count}}</span> people trying {{title}} right now!">
 <highlightPrivacy "Your privacy">
+
+<noScriptHeading "Uh oh...">
```

Comment:
Well, I mean, that's what we're de facto doing even by leaving the L20n stuff in there. It's just that some day we might be able to support L20n on the server side, too.